howto/install/deploy-juju.md

```diff
@@ -174,13 +174,13 @@ machines:
     constraints: "instance-type=g4dn.2xlarge root-disk=50G"
 ```
 
-To deploy with the bundle from above, add `--overlay overlay.yaml` to your deploy command:
+To deploy, add `--overlay overlay.yaml` to your deploy command. For example:
 
     juju deploy cs:~anbox-charmers/anbox-cloud ... --overlay overlay.yaml
 
-### Use Arm Instances
+### Use Arm instances
 
-Some clouds, like AWS with their Graviton instances, provide support for Arm instance types. These can be used with Anbox Cloud by specifying the correct instance type in the `overlay.yaml` as well:
+Some clouds, like AWS with their Graviton instances, provide support for Arm instance types. These can be used with Anbox Cloud by specifying the correct instance type in the `overlay.yaml`:
 
 ```
 applications:
```

Comment:
Question for line 188 below: I thought we don't support Juju 2.9, so why that comment - and does this syntax work with Juju 2.8?

Reply:
As of 1.12 we support 2.9 just fine. However not sure which comment you mean